opm/autodiff/SolventPropsAdFromDeck.cpp

```diff
@@ -274,6 +276,35 @@ SolventPropsAdFromDeck::SolventPropsAdFromDeck(DeckConstPtr deck,
                 }
             }
 
+            if (deck->hasKeyword("TLPMIXPA")) {
+                const TableContainer& tlpmixparTables = tables->getTlpmixpaTables();
+                if (!tlpmixparTables.empty()) {
+
+                    int numRegions = tlpmixparTables.size();
+                    // resize the attributes of the object
+                    tlpmix_param_.resize(numRegions);
+                    for (int regionIdx = 0; regionIdx < numRegions; ++regionIdx) {
+                        const Opm::TlpmixpaTable& tlpmixparTable = tlpmixparTables.getTable<TlpmixpaTable>(regionIdx);
+
+                        // Copy data
+                        const auto& po = tlpmixparTable.getOilPhasePressureColumn();
+                        const auto& tlpmixpa = tlpmixparTable.getMiscibilityColumn();
+
+                        tlpmix_param_[regionIdx] = NonuniformTableLinear<double>(po, tlpmixpa);
+
+                    }
+                } else {
+                    // if empty keyword. Try to use the pmisc table as default.
```

Comment:
Would this be correct? Is not the PMISC already accounted for, so it will be used twice?